tests/integration/test_end_to_end_election.py

```diff
@@ -166,71 +168,103 @@ def step_1_key_ceremony(self) -> None:
         # Setup Guardians
         for i in range(self.NUMBER_OF_GUARDIANS):
             self.guardians.append(
-                Guardian("guardian_" + str(i), i, self.NUMBER_OF_GUARDIANS, self.QUORUM)
+                Guardian(
```

Comment:
i know this was here before but it seems error prone and we've hit this before a few times when the indexing is off.  maybe we should move it into the key ceremony helper and reference any iterators that consume this pattern from there?  or is this part of #336 ?

Reply:
Actually this is a mistake. This should use the KeyCeremonyHelper to create the guardians. That'll reduce the need for #336.